noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -132,8 +132,10 @@ global INITIALIZATION_SLOT_SEPARATOR: Field = 1000_000_000;
 global INITIAL_L2_BLOCK_NUM: Field = 1;
 global BLOB_SIZE_IN_BYTES: Field = 31 * 4096;
 global ETHEREUM_SLOT_DURATION: u32 = 12;
+global AZTEC_SLOT_DURATION: u32 = ETHEREUM_SLOT_DURATION * 1;
```

Comment:
What is the need for the * 1?

Reply:
Want the `AZTEC_SLOT_DURATION` to be a multiple of a `ETHEREUM_SLOT_DURATION` so just makes it very easy to change it to 3 to have the different duration 🤷 Is not really needed when the value is 1, but just kinda nice to showcase multiple I think.